src/diracx/routers/__init__.py

```diff
@@ -143,6 +143,20 @@ def create_app_inner(
     app.add_exception_handler(DiracError, dirac_error_handler)
     app.add_exception_handler(DiracHttpResponse, http_response_handler)
 
+    from fastapi.middleware.cors import CORSMiddleware
+
+    origins = [
+        "http://localhost:8000",
```

Comment:
Should we create an environment variable to provide the `CORSMiddleware` with the `origins`?

As this is a temporary solution (:crossed_fingers: ), I prefer asking.

Reply:
It's also possible to inject this in the ingress controller so we might be best of letting it be an option of the helm chart rather than DiracX itself. (Though the hack is good enough for now.)

Reply:
(i.e. nginx would include extra headers in the response)

Reply:
Yes you're right, I will try to set this up in the ingress controller, thanks!

Reply:
Is the `CORSMiddleware` still needed now?

Reply:
Not for now at least.